.github/workflows/golangci-lint.yml

```diff
@@ -16,8 +16,12 @@ jobs:
               with:
                 fetch-depth: 0
                 submodules: 'true'
+            - name: Set up LibGL, Mesa & X11 libraries
+              run: |
+                sudo apt-get --allow-releaseinfo-change update
+                sudo apt-get install -y libgtk-3-dev libasound2-dev libxxf86vm-dev
             - name: golangci-lint
               uses: golangci/golangci-lint-action@v2
               with:
                   # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-                  version: 'v1.38.0'
+                  version: 'latest'
```

Comment:
Just my view, but I'm not sure, that putting here 'latest' tag is ok, because new linter version could cause lin errors in code, which was clean in a previous versions.
I'd prefer to put here `1.39` and manually update it, when new linter is there.

Reply:
I agree